### PR TITLE
Footer hover effect

### LINKED ISSUE #1093: make contact us section have some sort of hover effect
With the introduction of the sponsor section all parts of the website have a hover effect to show a item is clickable but the contact us section does not. A possible effect could be a lift up effect for the social images and a underline for the text.
image

### PR DESCRIPTION
**Pull request by ww2497**
resolves #1093

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -452,7 +452,8 @@ <h2>contact us</h2>
             </div>
             <div>
                 <p>
-                    <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct | </a>
+                    <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct</a>
+                    |
```
Comment (peterkos) [inline index.html:456]: Can the `|` be wrapped in a `p` tag?

**Review by peterkos — CHANGES_REQUESTED** [review]
Nice change!
Although can we have the links on hover change to `$red` insteaad of transforming? Transforms might look better under block elements and we can use color for inline; I'm planning the same thing for the updated schedule design and is what the current schedule uses but a bit less

**Comment by ww2497** [comment]
> Nice change!
> Although can we have the links on hover change to `$red` insteaad of transforming? Transforms might look better under block elements and we can use color for inline; I'm planning the same thing for the updated schedule design and is what the current schedule uses but a bit less

Alright, I'll see how that effect looks. I did the translate effect because that's apparently what design wanted according to the issue.

File: index.html [hunk 2]
```diff
@@ -534,7 +534,8 @@ <h2>contact us</h2>
             </div>
             <div>
                 <p>
-                    <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct | </a>
+                    <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct</a>
+                    <span>|</span>
```
Comment (peterkos) [inline index.html:538]: Oh wait, I see, this is all wrapped in `p`, the `span` isnt needed! My b

File: sass/main.scss [hunk 3]
```diff
@@ -1147,6 +1148,22 @@ footer {
...
         }
+
+        a {
+            display: inline-block;
+            transition: 0.2s;
+
+            i {
+                transition: 0.2s;
+            }
+            &:hover {
```
Comment (peterkos) [inline sass/main.scss:1159]: nit: `&:` styles should go above any other nested CSS, so this should go above the `i` above

**Review by peterkos — CHANGES_REQUESTED** [review]
One other tweak: Translate effect for logos should be put back -- having logos change color on hover is a branding guideline issue!

**Review by peterkos — APPROVED** [review]
LGTM!

**Pull request merged**